src/model/application.model.ts

```diff
@@ -34,7 +35,9 @@ export interface ApplicationData {
     has_sold_land?: string;
     is_secure_register?: string;
     who_is_registering?: string;
+    company_profile_details?: ICompanyDetails;
```

Comment:
Correspondingly change this to CompanyProfile

Reply:
👍🏿